khmer/khmer_args.py

```diff
@@ -149,6 +311,7 @@ def calculate_tablesize(args, hashtype, multiplier=1.0):
 
 
 def create_nodegraph(args, ksize=None, multiplier=1.0):
+    args = check_fp_rate(args)
```

Comment:
I'm conflicted on how to manage the fp rate setting--Currently, all scripts default to an fp of 0.1 but that can be overidden via the --fp-rate arg, which is in the args object and is passed along.

If we want to actually pass in a default rate we'd have to add an arg to at least both create functions--Which isn't necessarily bad, but I'd like to avoid it if possible.

We could have the scripts instead modify the fp_rate attribute of the args... Though I don't feel like that's a good solution either